framework/src/main/java/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -195,9 +196,34 @@ private void viewpointAdaptExecutable(
             receiver = FlowExpressions.internalReprOf(factory, receiverTree);
         }
 
-        List<FlowExpressions.Receiver> argReceivers = new ArrayList<>(args.size());
-        for (ExpressionTree argTree : args) {
-            argReceivers.add(FlowExpressions.internalReprOf(factory, argTree));
+        List<FlowExpressions.Receiver> argReceivers = new ArrayList<>();
+        boolean varargs = false;
+        if (tree instanceof MethodInvocationTree) {
```

Comment:
Don't use `instanceof`, check the Tree.Kind: https://checkerframework.org/manual/#creating-using-the-apis.